doc/testing.md

```diff
@@ -135,10 +135,19 @@ the first N tiers they can afford to run, but at least tier1.
 
 A brief description of the tiered test groups:
 
-- `tier1`: This is the lowest test tier. Multiple developers run these tests
+- `tier1`: This is the most fundamental test tier.
+  Roughly speaking, a failure of a test in this tier has the potential to
+  indicate a problem that would affect many Java programs. Tests in `tier1` include
+  tests of HotSpot, core libraries in the `java.base` module, and the `javac` compiler.
```

Comment:
I think I'd tweak "core libraries in the java.base module" to say "core classes" or "core APIs", only because there isn't really any notion of a module containing libraries. If someone asks then they'll have to directed to the tier1 definition in test/jdk/TEST.groups as it's hard to come up with a one-line summary.